seed/seed.ts

```diff
@@ -1,16 +1,50 @@
 import { PrismaClient } from '@prisma/client'
-import seedRaces from './races.seed'
+import { parseArgs } from 'util'
+
+// factory seeds
+import seedRaces from './races'
 import seedCampaigns from './campaigns'
 import seedTopIssues from './topIssues'
 import seedUsers from './users'
 
+// csv file seeds
+import seedElectionTypes from './electionTypes'
+import seedMtfcc from './mtfcc'
+import seedCounties from './counties'
+import seedMunicipalities from './municipalities'
+
+const IS_PROD_BUILD =
+  process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'qa'
+
```

Comment:
I think we need to add `development` here as well so that the seeds don't wipe-out that DB as well

Reply:
Don't we need all the seeds to run on branch-preview envs tho? :thinking_face:

So yeah.. in prod, qa, and dev, we only want the "csv" seeds to run.. everywhere else, do it all..